### ISSUE TITLE
DiscordChatExplorer requires macOS 10.13+ but readme claims otherwise

### ISSUE THREAD

**Issue by andlabs** [Tyrrrz/DiscordChatExporter#551]
I can't patch this myself without going through a process with my employer and I'm not sure if you want to just drop macOS 10.11/10.12 or downgrade the version of .net Core you use, but .net Core 3.1 requires macOS 10.13, which contradicts your README saying DiscordChatExplorer requires macOS 10.11.

The `dotnet` executable and the help system will run, but starting the archive process will result in

```
Exporting 127 channels... 0.00%dyld: lazy symbol binding failed: Symbol not found: _utimensat
  Referenced from: /Users/pietro/extsrc/dotnet-runtime-3.1.14-osx-x64/shared/Microsoft.NETCore.App/3.1.14/System.Native.dylib
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _utimensat
  Referenced from: /Users/pietro/extsrc/dotnet-runtime-3.1.14-osx-x64/shared/Microsoft.NETCore.App/3.1.14/System.Native.dylib
  Expected in: /usr/lib/libSystem.B.dylib

Abort trap: 6
134
```

with `utimesat()` not existing before 10.13 and the lack of support being intentional — https://github.com/dotnet/core/issues/4005. (And I'm choosing to stay on 10.12 because I don't want Disk Utility converting my backup drives to APFS without my permission.)

Thanks!

**Comment by Tyrrrz** [comment]
Thanks for bringing it to my attention. I don't use macOS myself, so I referenced the docs to see which versions are supported.
I'm not sure where I got 10.11+ from because neither 10.11 nor 10.12 are mentioned as supported by .NET 3.1 in the docs:

https://docs.microsoft.com/en-us/dotnet/core/install/macos

Going forward there are no plans to downgrade, in fact I'd want to migrate to .NET 6.0 once it comes out and I'm able to move users to that version seamlessly: #521.

However, you may still be able to use DCE on older versions of macOS by running it as a Docker container. This way you won't have to install any prerequisites on your host machine and it won't matter what version of macOS you are using. https://hub.docker.com/r/tyrrrz/discordchatexporter

For now, I will fix the readme to say "macOS 10.13+" instead.

**Comment by andlabs** [comment]
All right; thanks!
